src/algorithms.jl

```diff
@@ -847,8 +847,16 @@ SKenCarp(;chunk_size=0,autodiff=true,diff_type=Val{:central},
 
 # Jumps
 
-struct TauLeaping <: StochasticDiffEqJumpAdaptiveAlgorithm end
-struct CaoTauLeaping <: StochasticDiffEqJumpAdaptiveAlgorithm end
+struct TauLeaping <: StochasticDiffEqJumpAdaptiveAlgorithm
+  adaptive::Bool
+end
+TauLeaping(; adaptive=true) = TauLeaping(adaptive)
+
+struct CaoTauLeaping <: StochasticDiffEqJumpAdaptiveAlgorithm
```

Comment:
add the reference to the docstring. See OrdinaryDiffEq for how this is usually done